docs/topics/feed-exports.rst

```diff
@@ -268,6 +268,26 @@ in multiple files, with the specified maximum item count per file. That way, as
 soon as a file reaches the maximum item count, that file is delivered to the
 feed URI, allowing item delivery to start way before the end of the crawl.
 
+.. _item-filter:
+
+Item filter
+===========
+
+An `ItemChecker` class can be used to create your own custom item filter.
+Creating such an item filter can help filter scraped items based on some custom
+conditions that you can write in the `accepts` methods of the class. The accepts
+method must return a bool value to determine if the item is acceptable or not.
+These filters can then be activated by declaring them in :ref:`feeds options <feed-options>`.
+
+For instance::
+
+    class MyCustomFilter(scrapy.itemchecker.ItemChecker):
+
+        def accepts(self, item):
+            if "field1" in item and item["field1"] == "expected_data":
+                return True
+            return False
+
```

Comment:
I would call the section “Item filtering” instead, so that it can be about item filtering in general, and not specifically about the item filter class.

Then you could start the section by explaining that Scrapy allows to use the `item_classes` option in `FEEDS` to indicate which types of items to deliver to each feed. Then you could continue by explaining that this feature is implemented by Scrapy’s `ItemChecker` class, which is the default value of the `item_filter` option in `FEEDS`. Then  you could go on to explain how to create custom item filtering classes.

I also think that it may be best to rename `scrapy.itemchecker.ItemChecker` to `scrapy.itemfilter.ItemFilter`, for consistency. In fact, it may be best to move it to `scrapy.extensions.feedexport.ItemFilter`, for consistency with other classes of the extension.

Finally, it would be great to include here at the end the reference documentation of the `ItemChecker` class, with `.. autoclass: scrapy.itemchecker.ItemChecker`. You can then use ``` :class:`~scrapy.itemchecker.ItemChecker` ``` instead of ``` `ItemChecker` ``` when referring to the class from other parts of the documentation.